harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/geonet/Aligner.java

```diff
@@ -166,6 +166,7 @@ public HarvestResult align(Set<RecordInfo> records, List<HarvestError> errors) t
 		//--- remove old metadata
 
 		for (String uuid : localUuids.getUUIDs()) {
+		    //FIXME can we do it on one step?
```

Comment:
It was a note to remind myself to check about this. If we can do the remove old metadata in one step instead of a loop... but I couldn't think of a clean way to do that right. I just wanted the harvester to be scalable even if you handle a lot of records.